go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/go-git/go-billy/v5
+module github.com/c4milo/go-billy/v5
```

Comment:
Obviously this is a no-go. I would suggest creating this PR from a different branch in your fork, so you can keep your master importable while not polluting the PR.

Reply:
yes, sorry about that.